packages/amplication-client/src/Entity/EntityFieldListItem.scss

```diff
@@ -8,17 +8,32 @@
     @include flexFullRowWithSpacing;
     margin-bottom: var(--default-spacing);
 
+    &:nth-child(2) {
```

Comment:
I would prefer to add a specific css to this child instead of its position.

Reply:
Added a new class to the second child to add border-bottom styles(  `&__description-row` ) ✅